sdk/language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/helpers/TransactionGenerator.scala

```diff
@@ -257,9 +257,10 @@ object TransactionGenerator {
     contractId <- nonEmptyId
     (scalaTemplateId, javaTemplateId) <- identifierGen
     parties <- Gen.listOf(nonEmptyId)
+    packageName <- Gen.option(nonEmptyId)
   } yield (
     Archived(ArchivedEvent(eventId, contractId, Some(scalaTemplateId), parties)),
-    new data.ArchivedEvent(parties.asJava, eventId, javaTemplateId, contractId),
+    new data.ArchivedEvent(parties.asJava, eventId, javaTemplateId, contractId, packageName.toJava),
   )
 
   val exercisedEventGen: Gen[(Exercised, data.ExercisedEvent)] = for {
```

Comment:
Thanks - line 297 also needs to be patched according to:

```
language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/helpers/TransactionGenerator.scala:297: error: not enough arguments for constructor ExercisedEvent: (witnessParties: java.util.List[String], eventId: String, templateId: com.daml.ledger.javaapi.data.Identifier, interfaceId: java.util.Optional[com.daml.ledger.javaapi.data.Identifier], contractId: String, choice: String, choiceArgument: com.daml.ledger.javaapi.data.Value, actingParties: java.util.List[String], consuming: Boolean, childEventIds: java.util.List[String], exerciseResult: com.daml.ledger.javaapi.data.Value, choicePackageId: java.util.Optional[String]): com.daml.ledger.javaapi.data.ExercisedEvent.
Unspecified value parameter choicePackageId.
    new data.ExercisedEvent(
    ^
```

https://dev.azure.com/digitalasset/daml/_build/results?buildId=178552&view=logs&jobId=82d7f739-b426-56b1-5402-5fa912747784&j=2e1b8928-2d4f-5c49-dab1-f9c1bbd1b976&t=799d407b-af82-5eb5-b898-1d5c693796b3

Reply:
Ah that was a separate PR: https://github.com/DACH-NY/canton/commit/9fa27813b695b0f20412816dac4787ea4c3d9ba6